src/cascadia/TerminalControl/IControlSettings.idl

```diff
@@ -57,5 +57,7 @@ namespace Microsoft.Terminal.TerminalControl
         Boolean RetroTerminalEffect;
         Boolean ForceFullRepaintRendering;
         Boolean SoftwareRendering;
+
+        Boolean DisableProgressRing;

```

Comment:
So, this setting is specific to _TerminalApp_ .. it isn't something that Control needs to know about. We shouldn't be using IControlSettings as a way for TerminalApp to _tell itself something it could find out internally_ (we're asking TerminalControl to hold on to a value that only TA cares about.)